code/game/atoms.dm

```diff
@@ -144,9 +145,12 @@
 		alternate_appearances = null
 
 	QDEL_NULL(reagents)
-	invisibility = 101
+	invisibility = INVISIBILITY_MAXIMUM
 	LAZYCLEARLIST(overlays)
 	LAZYCLEARLIST(priority_overlays)
+
+	QDEL_NULL(light)

```

Comment:
I'm kinda surprised we never did this before. granted. I believe everything had `set_light(0)` somewhere, but this is still odd this wasn't a thing.

Reply:
Aye, very strange indeed.